include/highfive/bits/H5Node_traits_misc.hpp

```diff
@@ -97,14 +97,14 @@ NodeTraits<Derivate>::createDataSet(const std::string& dataset_name,
 template <typename Derivate>
 inline DataSet
 NodeTraits<Derivate>::getDataSet(const std::string& dataset_name,
-                                 const DataSetAccessProps& accessProps) const {
-    const auto hid = H5Dopen2(static_cast<const Derivate*>(this)->getId(),
-                              dataset_name.c_str(), accessProps.getId());
-    if (hid < 0) {
-        HDF5ErrMapper::ToException<DataSetException>(
-            std::string("Unable to open the dataset \"") + dataset_name + "\":");
-    }
-    return DataSet(hid);
+                                const DataSetAccessProps& accessProps) const {
+   const auto hid = H5Dopen2(static_cast<const Derivate*>(this)->getId(),
```

Comment:
I guess this reformat was a mistake. large diff and against our coding style... We can't accept changes in this state.